webapp-next/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM centos:8
```

Comment:
Can `registry.access.redhat.com/ubi8/ubi-minimal` be used instead (as in other services)?